frontend/src/components/userPage/UserPageContent.jsx

```diff
@@ -103,6 +116,18 @@ const UserPageContent = () => {
         }
     }
 
+    const handleChangeLocation = async () => {
+        try {
+            await axios.patch(`${url}/users/location`, {
+                lat: userPosition.latitude,
+                long: userPosition.longitude,
+            })
+            location.reload()
```

Comment:
Why is reload needed here? Isn't your userPosition state already up-to-date?

Reply:
Yes, it is

Reply:
To do:
Fix modify user position UserPageContent.jsx

Reply:
Done